### ISSUE TITLE
Rating - Provide alternative text for images - (2036570025)

### ISSUE THREAD

**Issue by dqateam** [Esri/calcite-components#5588]
### Module: 
 20b Rating 

### Violation: 
 Provide alternative text for images 

![image](https://user-images.githubusercontent.com/5023024/202751523-40f1f0b3-bc0a-41a6-b368-602cffab8b24.png)

### WCAG Reference: 
- [1.1.1 Non-text Content](https://www.w3.org/WAI/WCAG21/Understanding/non-text-content.html) 

### Instance ID: 
 2036570025 

### Severity: 
 10 

### Description: 
 [Issue]
There are meaningful Star rating images without a textual equivalent. which are not announced to the screen reader users. 

[User Impact]
Screen reader users will be unable to determine what these images represent.

[Code Reference] 
 ```
<span class="wrapper"><label class="star average" for="calcite-ratings-3ab75500-71fa-d6c0-de27-427adf4ab2ba-1">
<calcite-icon aria-hidden="true" class="icon" icon="star-f" scale="m" calcite-hydrated=""></calcite-icon>
<svg class="svg" fill="currentColor" height="100%" viewBox="0 0 24 24" width="100%" xmlns="http://www.w3.org/2000/svg">
...
</svg>
 ``` 

### Note: 
 [Suggestion]
Provide alternative text for images. Meaningful images must have a concise but descriptive textual equivalent. 

In this case provide role="img" and title attribute to the div element containing star images.

[Compliant Code Example] 
 ```
<span class="wrapper"><label class="star average" for="calcite-ratings-3ab75500-71fa-d6c0-de27-427adf4ab2ba-1">
<calcite-icon aria-hidden="true" class="icon" icon="star-f" scale="m" calcite-hydrated=""></calcite-icon>
<svg role="img" aria-label="Rated 4 out of 5" class="svg" fill="currentColor" height="100%" viewBox="0 0 24 24" width="100%" xmlns="http://www.w3.org/2000/svg">
...
</svg>
 ```
 

### Media Type: 
 Images 

### Additional Resources: 
- Report Source: Product accessibility evaluation conducted on the Calcite Design System in August, 2022.

**Comment by SaraLawrence** [comment]
@geospatialem I totally agree with you. The tester is not acknowledging the context provided and also the fact that these are SVGs with valid aria-labels.

**Comment by geospatialem** [comment]
Awesome, thanks much! Going to close this one out. 👍🏻
